core/src/main/java/com/nukesz/game/BodyPart.java

```diff
@@ -3,24 +3,35 @@
 import com.badlogic.gdx.graphics.Texture;
 import com.badlogic.gdx.graphics.g2d.Batch;
 
-public class BodyPart {
-    public int x;
-    public int y;
-    private Texture texture;
+public class BodyPart extends Item {
+
 
     public BodyPart(Texture texture) {
-        this.texture = texture;
+        super(0, 0, texture);
+    }
+
+    public BodyPart(int x, int y) {
+        super(x, y, null);
     }
 
     public void updateBodyPosition(int x, int y) {
         this.x = x;
         this.y = y;
     }
 
+    public int getX() {
+        return x;
+    }
+
+    public int getY() {
```

Comment:
in libgdx, they are not really adding getters and setters. They prefer to have public fields. It's not just performance, but usually a class can have 5-10+ fields, and a setters and getters adding noise around it.  

It's easier to write 
car.speed += 10;

then
car.setSpeed(car.getSpeed() + 10);